### PR TITLE
Package pyml_bindgen.0.3.1

### PR DESCRIPTION
**Pull request by mooreryan**
### `pyml_bindgen.0.3.1`
Generate pyml bindings from OCaml value specifications

---
* Homepage: https://github.com/mooreryan/ocaml_python_bindgen
* Source repo: git+https://github.com/mooreryan/ocaml_python_bindgen.git
* Bug tracker: https://github.com/mooreryan/ocaml_python_bindgen/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/pyml_bindgen/pyml_bindgen.0.3.1/opam [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+bug-reports: "https://github.com/mooreryan/ocaml_python_bindgen/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "angstrom" {>= "0.15.0"}
+  "base" {>= "v0.12"}
+  "cmdliner" {>= "1.1.0"}
+  "ppx_let" {>= "v0.12"}
+  "ppx_sexp_conv" {>= "v0.12"}
+  "ppx_string" {>= "v0.12"}
+  "re" {>= "1.5"}
```
Comment (mseri) [inline packages/pyml_bindgen/pyml_bindgen.0.3.1/opam:17]:
```suggestion
  "re" {>= "1.7.2"}
```
Solves `Unbound module Re.Perl`. This is the version that introduced the `Re.` modules in place of `Re_`

**Comment by mooreryan** [comment]
Oh and it looks like it's still failing on the x86_32 ...I guess we should still mark it as unavailable on that platform?

**Comment by mseri** [comment]
No, that is not a real failure, it is due to a compiler relocation warning on x86_32

File: packages/pyml_bindgen/pyml_bindgen.0.3.1/opam [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+bug-reports: "https://github.com/mooreryan/ocaml_python_bindgen/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "angstrom" {>= "0.15.0"}
+  "base" {>= "v0.12"}
+  "cmdliner" {>= "1.1.0"}
+  "ppx_let" {>= "v0.12"}
+  "ppx_sexp_conv" {>= "v0.12"}
+  "ppx_string" {>= "v0.12"}
+  "re" {>= "1.7.2"}
```
Comment (mseri) [inline packages/pyml_bindgen/pyml_bindgen.0.3.1/opam:17]:
```suggestion
  "re" {>= "1.10.0"}
```
to fix 
```
Unbound value Re.Group.get_opt
```

**Comment by mseri** [comment]
Can you please upstream the lower bound change?

**Pull request merged**